sgx_urts/src/fd.rs

```diff
@@ -217,6 +219,23 @@ pub extern "C" fn u_fcntl_arg1_ocall(
     ret
 }
 
+#[cfg(all(target_env="musl", not(target_env="")))]
+#[no_mangle]
+pub extern "C" fn u_ioctl_arg0_ocall(error: * mut c_int,
+                                     fd: c_int,
+                                     request: c_int) -> c_int {
+    let mut errno = 0;
+    let ret = unsafe { libc::ioctl(fd, request) };
```

Comment:
I suggest adding conditional compilation option in this line `(let ret = unsafe { libc::ioctl(fd, request) }).` Because the function prototype has not changed, I think there is no need to define a function for the musl target separately.